### DISCUSSION TITLE
sveltekit authjs session update

### DISCUSSION THREAD

**Discussion by AV6** [nextauthjs/next-auth#9515] [Help]
How does one update the session created from authjs (not from the session callback) at a random point in time from one of the api routes for example? is there an authjs function to do it or are we expected to handle the cookies explicitely?

**Comment by oliverjohns** [comment]
I also wonder this. In NextJS version of AuthJS there is [useSession](https://next-auth.js.org/getting-started/client#updating-the-session), but not in Sveltekit it seems?
